docs/diagrams/AttendanceClassDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+!include style.puml
+show members
+hide empty members
+skinparam classAttributeFontColor #FFFFFF
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+Package Attendance as AttendancePackage <<Rectangle>>{
+
+Class UniqueAttendanceList
+Class Attendance
+Class Hours
+Class Name
+Class Remark
+}
+
+Class HiddenOutside #FFFFFF
+HiddenOutside ..> UniqueAttendanceList
+
+UniqueAttendanceList ..> "*" Attendance
+Attendance *---> "Member 1" Name
+Attendance *---> "1 Event" Name
```

Comment:
Should 'Member' and 'Event' be 'MemberName' and 'EventName' to be consistent with the attributes in attendance?

Reply:
As this is a class diagram, and there are no `EventName` or `MemberName` classes, I think we should keep it to `Member` and `Event`